docroot/themes/custom/yukonca_glider/src/sass/pages/_site-wide-alert.page.scss

```diff
@@ -0,0 +1,5 @@
+.node--type-site-wide-alert {
+  .callout-card__content-wrapper {
+    @apply lg:ml-[25%];
```

Comment:
Can't we move this part to a component class?

Reply:
It is actual ONLY for Site wide alert template - not for general